### DISCUSSION TITLE
Add schools

### DISCUSSION THREAD

**Discussion by savalet** [community/community#31224] [GitHub Education]
A

**Comment by samus-aran** [comment]
We are happy you are here! 👋  You are more likely to get a useful response if you are explicit about what your query entails, giving a few more details might help someone give you a nudge in the right direction.

**Comment by khuwender** [comment]
[Ezyschooling](https://ezyschooling.com) is an online platform that helps parents find and apply to the [best boarding schools in India](https://ezyschooling.com/admissions/school-admission-in-boarding-schools). The platform has a database of over 1500 schools, including both public and private schools. Parents can use the platform to search for schools by location, curriculum, fees, and other factors. Ezyschooling also provides information about the application process, admission requirements, and financial aid.
